docs/usage/configuration/config_documentation.md

```diff
@@ -462,6 +462,21 @@ See the docs [request log format](../administration/request_log.md).
 * `additional_resources`: Only valid for an 'http' listener. A map of
    additional endpoints which should be loaded via dynamic modules.
 
+  _Added in Synapse 1.86.0_
```

Comment:
Currently, this gets grouped to the list item above. It should be added below the item or perhaps in parenthesis next to `Unix socket support (...):`

Reply:
The formatting leaves much to be desired. Look right above this section and you'll see a subsection(ish) that is for 'Added in Synapse 1.68.0' that blends in too. There are a bunch of spots where you can find various 'Added in Synapse x.xx.x' that look like this

Reply:
Actually, I wonder if I stick 4 `#`'s in front of it how it would look(Pycharm's render mode is only slightly helpful). Let's try it.

Reply:
https://pr15708--synapse-docs-previews.netlify.app/usage/configuration/config_documentation.html#listeners

Well, that's a little bolder I guess. What do you think?

Reply:
The problem is that it's grouped in the `<li>` above where it doesn't apply.

The look could be improved but it aligns with how we do it for the rest of the things there so it's fine.

Reply:
Oh. I thought it was ok like this as it was a 'feature' of the `listeners` config section.

Reply:
I've thought about the potential of issues with using 4 `#`'s and decided it's not worth that, so I'll abandon that line of thought. Let's continue this with the other comment thread, as it's more relevant there.